src/styles/main.scss

```diff
@@ -1,3 +1,53 @@
+@import "variables";
+
+html {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
 body {
   margin: 0;
 }
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $space-size;
+  width: $calendar-size;
+  padding: $calendar-padding;
+  margin: 0 auto;
+
+  &__day {
+    @include cell-propertis(100px, 100px, #eee);
+    border: 1px solid black;
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    transition: background-color 0.5s, transform 0.5s;
+    cursor: pointer;
+  }
+
+  &__day:hover {
+    background-color: $color-hover;
+    transform: translateY(-20px);
+  }
+}
+
+@for $i from 1 through 31 {
+  .calendar__day:nth-child(#{$i})::before {
```

Comment:
Don't forget about nesting and use it wherever possible.
https://sass-lang.com/guide/#nesting